app/src/main/java/org/oppia/android/app/customview/SegmentedCircularProgressView.kt

```diff
@@ -109,12 +124,20 @@ class SegmentedCircularProgressView : View {
         canvas.drawArc(baseRect!!, startAngle, sweepAngle, false, chapterFinishedArcPaint)
```

Comment:
can we use lateinit instead of making it nullable?

Reply:
done